grid2op/tests/test_alert.py

```diff
@@ -13,45 +13,1234 @@
+    def test_assistant_reward_value_blackout_2_lines_attacked_simulaneous_only_1_alert(self) -> None:
+        """
+        When 2 lines are attacked simustaneously at step 2 and we raise only 1 alert 
+        we expect a reward of -4 at step 3 
+        """
+        kwargs_opponent = dict(lines_attacked=[ATTACKED_LINE]+['48_53_141'], 
+                                   duration=3, 
+                                   steps_attack=[2,2])
+        with make(self.env_nm, test=True, difficulty="1", 
+                            opponent_attack_cooldown=0, 
+                            opponent_attack_duration=99999, 
+                            opponent_budget_per_ts=1000, 
+                            opponent_init_budget=10000., 
+                            opponent_action_class=PlayableAction, 
+                            opponent_class=TestOpponent, 
+                            kwargs_opponent=kwargs_opponent
+            ) as env : 
+            new_param = Parameters()
+            new_param.MAX_LINE_STATUS_CHANGED = 10
+
+            env.change_parameters(new_param)
+            env.seed(0)
+            env.reset()
+            for i in range(env.max_episode_duration()):
+                attackable_line_id = 0
+                act = self.do_nothing
+                if i == 3 : 
+                    act = self.blackout_action
+                elif i == 1:
+                    act = self.env.action_space({"raise_alert": [0]})
+                obs, reward, done, info = env.step(act)
+                if i == 2 : 
+                    assert env._oppSpace.last_attack is not None
+                
+                if i == 3: 
+                    assert reward == -4
+                else : 
+                    assert reward == 0
+
+                if done: 
+                    break
+
+# return 2
+    def  test_assistant_reward_value_blackout_2_lines_different_step_in_window_good_alerts(self) -> None : 
+        """
+        When 2 lines are attacked at different steps 2 and 3 and we raise 2  alert 
+        we expect a reward of 2 at step 3 
+        """
+        kwargs_opponent = dict(lines_attacked=[ATTACKED_LINE]+['48_53_141'], 
+                                   duration=[1,1], 
+                                   steps_attack=[2, 3])
+        with make(self.env_nm, test=True, difficulty="1", 
+                            opponent_attack_cooldown=0, 
+                            opponent_attack_duration=99999, 
+                            opponent_budget_per_ts=1000, 
+                            opponent_init_budget=10000., 
+                            opponent_action_class=PlayableAction, 
+                            opponent_class=TestOpponentMultiLines, 
+                            kwargs_opponent=kwargs_opponent,
+                            reward_class=AlertReward(reward_end_episode_bonus=42)
+            ) as env : 
+            env.seed(0)
+            env.reset()
+            for i in range(env.max_episode_duration()):
```

Comment:
can you remove this "for" loop and make the thing sequential (you only need to do 3 actions, 3 steps and 3 checks, it's clearer without the for loop I think)